### PR TITLE
Remove KDE dependency, add qmake projects

### PR DESCRIPTION
**Pull request by martonmiklos**
This PR will remove the KDE dependencies. There is a functionality loss around the kodemain where the creator e-mail address/name needs to be passed as an argument not retrived from the address book. 

This PR also adds pro files to be able to build the project with qmake.

It also patches the cmake files to use Qt5. If I remember well we have discussed the Qt4 dropping earlier, but I have not been able to find your answer now. I am under the impression that Qt4 is now far "away", but I would like to hear your opinion. If you do not like dropping Qt4 I can do some research how can cmake handle both versions. (I have seen such a thing in ROS stuff, so it is possible).

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,38 +1,58 @@
+cmake_minimum_required(VERSION 3.5)
+
 project( kode )
-set(CMAKE_MODULE_PATH "${kode_SOURCE_DIR}/cmake/modules")
 
-find_package(KDE4 REQUIRED)
```
Comment (dfaure-kdab) [inline CMakeLists.txt]: The commit message is doubly misleading :-)
1) this was depending on KDE4 (kdelibs4 in fact), not 5
2) there's no such thing as "KDE5" anyway (there's KDE Frameworks 5, KDE Plasma 5, and apps...)

Comment (martonmiklos) [reply CMakeLists.txt]: Hi David,
Thanks for the feedback, I have fixed title.

**Comment by dfaure-kdab** [comment]
I don't mind, I have my own copy of libkode in KDSoap, and I build it with both qmake and cmake.

**Comment by martonmiklos** [comment]
@cornelius 
I am ok with having push access to the repo, I will polish my git skills to make less mess than I used to do with the commits ;).

**Comment by cornelius** [comment]
@martonmiklos ok, I sent you an invite. Have fun :smile:

**Pull request merged**